tests/admin-ui-tests/list-view-crud.test.ts

```diff
@@ -0,0 +1,126 @@
+import { Browser, Page } from 'playwright';
+import fetch from 'node-fetch';
+import { adminUITests } from './utils';
+
+adminUITests('./tests/test-projects/crud-notifications', (browserType, deleteAllData) => {
+  let browser: Browser = undefined as any;
+  let page: Page = undefined as any;
+  const seedData = async (page: Page, query: string, variables?: Record<string, any>) => {
+    try {
+      const { errors } = await fetch('http://localhost:3000/api/graphql', {
```

Comment:
we were doing things like this previously 
```
await page.evaluate((query, variable) => {
   return fetch('http://localhost:3000/api/graphql', {
        method: 'POST',
        headers: {
          'Content-Type': 'application/json',
        },
        body: JSON.stringify({
          query,
          variables,
        }),
      }).then(res => res.json());
}, { query, variable)
```
but I ripped that out in favor of node-fetch, as we were getting a few rather non-deterministic network issues with this method, (most likely due to shifting network contexts from where page.evaluate is called) 

Running requests to api from node seems like the better more deterministic choice here, but I'm happy to re-evaluate this choice if i'm missing something here.